new_lineage_panel/src/App.tsx

```diff
@@ -409,7 +395,10 @@ function App() {
           sidebarScreen,
           setSidebarScreen,
           selectedColumn,
-          setSelectedColumn,
+          setSelectedColumn: (v) => {
+            setSelectedColumn(v);
+            sessionId = uuidv4();
```

Comment:
this one too. better to be a state/ref/context variable instead of static reference